src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -64,8 +63,12 @@ private static PowerFxConfig AddInterpreterFunctions(PowerFxConfig powerFxConfig
         private protected override INameResolver CreateResolver(PowerFxConfig alternateConfig = null)
         {
             // The RecalcEngineResolver allows access to the values from UpdateValue. 
-            var resolver = new RecalcEngineResolver(this, alternateConfig ?? Config);
-            return resolver;
+            return new RecalcEngineResolver(this, alternateConfig ?? Config);          
+        }
+
+        private protected override INameResolver CreateResolver(PowerFxConfig alternateConfig, IReadOnlyDictionary<string, NameLookupInfo> globalSymbols)

```

Comment:
>, IReadOnlyDictionary<string, NameLookupInfo> globalSymbols [](http://example.com/codeflow?start=93&length=59)

Do we need to pass this in as a parameter (and hence add a new virtual ) 

This is just computed from this.Formula, which we already have on this instance.